docs/source/playbook.rst

```diff
@@ -0,0 +1,191 @@
+Writing actions
+===============
+
+All Ansible is contained in `obal/data`. There we have `playbooks` and `roles`.
+
+A `playbook` with `metadata` is considered an action and exposed to the user as such.
+
+Writing playbooks
+-----------------
+
+We have a slightly non-standard playbooks layout. Every playbook is contained in its own directory and named `playbook.yaml`. It can also contain a `metadata.yaml`. While playbooks are pure ansible, the metadata is the data Obal needs to extract to build a CLI.
+
+Obal uses "hosts" to operate on. If you have a set of packages then every package is a host. There are some special hosts:
```

Comment:
I would argue that Obal uses an inventory to operate on. Obal's inventory is a set of packages. Introducing host I think may add confusion and deter from what we really want -- which is to get users to think of things as simply an inventory.

Reply:
It's not lost on me that the term hosts is unavoidable in a few places such as playbooks, but if we are talking about user mentality, I want users to think about inventory.